nodejs/eks/cluster.ts

```diff
@@ -371,6 +380,17 @@ export interface ClusterOptions {
      */
     instanceRole?: pulumi.Input<aws.iam.Role>;
 
+    /**
+     * Node instance profile - for worker nodes. If not supplied, it will be created.
+     */
+    instanceProfileName?: string;
+
+    /**
+     * Service role for eks cluster
+     *
```

Comment:
```suggestion
     *IAM Service Role for EKS to use to manage the cluster.
```

Reply:
replace both lines w/ the suggestion - I guess you can only make suggested changes to a single line instead of two.